Client/Frontend/Browser/SearchEngines.swift

```diff
@@ -118,13 +118,36 @@ class SearchEngines {
         disabledEngineNames[engine.shortName] = true
     }
 
+    func reloadEngines() {
+        self.orderedEngines = getOrderedEngines()
+    }
+
+    func deleteEngine(engine: OpenSearchEngine) {
```

Comment:
I'm getting a crash whenever I try to delete a custom engine. Do you see this too?